plugins/schedule/base/lib/model/ScheduleEvent.php

```diff
@@ -314,5 +314,20 @@ public function getSphinxMatchOptimizations() {
 		$objectName = $this->getIndexObjectName();
 		return $objectName::getSphinxMatchOptimizations($this);
 	}
+
+	public function isTimeInEvent($epochTime)
+	{
+		$startTs = $this->getStartTime();
+		$endTs = strtotime($this->end_date);

```

Comment:
Need to have equality , if start time get his own function to get the time in correct formal , end time needs one also